iroh-dag-sync/src/sync.rs

```diff
@@ -77,8 +77,24 @@ where
             send.0
                 .write_all(&SyncResponseHeader::Data(hash).as_bytes())
                 .await?;
-            send_blob::<iroh_blobs::store::fs::Store, _>(blobs, hash, &RangeSpec::all(), &mut send)
-                .await?;
+
+            // TODO(ramfox): not exactly sure what this should be
+            // Would be nice to have this be optional, or to have an empty Event
+            let mk_progress = |end_offset| iroh_blobs::provider::Event::TransferProgress {
+                connection_id: 0,
+                request_id: 0,
+                hash,
+                end_offset,
+            };
+            send_blob::<iroh_blobs::store::fs::Store, _>(
+                blobs,
+                hash,
+                &RangeSpec::all(),
+                &mut send,
+                EventSender::new(None),
+                mk_progress,
+            )
+            .await?;
```

Comment:
@dignifiedquire @rklaehn 

Hi, with the `send_blob` API changing, I'm not totally sure how to adjust this method. Do we _need_ to pass in a function?